docs/FAQ.md

```diff
@@ -111,6 +111,79 @@ store arbitrary files in `<your-git-repo>/scratch/`.
 
 You can find more details on `gitignore` files [here][gitignore].
 
+### How can I avoid committing my local-only changes to tracked files?
+
+Suppose your repository tracks a file like `secret_config.json`, and you make
+some changes to that file to work locally. Since Jujutsu automatically commits
+the working copy, there's no way to prevent Jujutsu from committing changes to
+the file. But, you never want to push those changes to the remote repository. 
+
+One solution is to keep these changes in a separate commit branched from the
+trunk and _merge_ the private commit into your branch to use those changes.
+
+Suppose you have a commit "Add new feature":
+
+```shell
+$ jj log
+@  mmzpqlum me@example.com 2024-08-21 11:13:21 ef612875
+│  Add new feature
+◉  tvxsnxqm me@example.com 2024-08-21 11:13:09 main b624cf12
+│  Existing work
+~
+```
+
+First, create a new commit branched from main and add your private changes:
+
+```shell
+$ jj new main -m "private: my credentials"
+Working copy now at: ttsqqnrx 861de9eb (empty) private: my credentials
+Parent commit      : tvxsnxqm b624cf12 main | Existing work
+Added 0 files, modified 1 files, removed 0 files
+
+$ echo '{ "password": "p@ssw0rd1" }' > secret_config.json
+```
+
+Now create a merge commit with the branch you're working on and the private
+commit:
+
+```shell
+$ jj new mmzpqlum ttsqqnrx
+Working copy now at: wktxklmk ac4d9fbe (empty) (no description set)
+Parent commit      : mmzpqlum ef612875 Add new feature
+Parent commit      : ttsqqnrx 2106921e private: my credentials
+Added 0 files, modified 1 files, removed 0 files
+
+$ jj log
+@    wktxklmk me@example.com 2024-08-22 08:57:40 ac4d9fbe
+├─╮  (empty) (no description set)
+│ ◉  ttsqqnrx me@example.com 2024-08-22 08:57:40 2106921e
+│ │  private: my credentials
+◉ │  mmzpqlum me@example.com 2024-08-21 11:13:21 ef612875
+├─╯  Add new feature
+◉  tvxsnxqm me@example.com 2024-08-21 11:13:09 main b624cf12
+│  Existing work
+~
+```
+
+The revision _ttsqqnrx_ contains changes from both _mmzpqlum_ and _ttsqqnrx_. As
```

Comment:
Nit: This is imo, to short of a explainer, see my suggestion for clarification.

Reply:
I added some bullet points here, does that help?

I also edited the change IDs to _yyyyyyy_, _xxxxxxx_, etc., to help visually distinguish them in the paragraphs.